samples/apps/copilot-chat-app/webapp/src/components/chat/ChatWindow.tsx

```diff
@@ -80,6 +86,19 @@ const useClasses = makeStyles({
         alignSelf: 'end',
         ...shorthands.gap(tokens.spacingVerticalS),
     },
+    content: {
+        ...shorthands.gridArea('content'),
+        overflowY: 'auto',
+    },
+    contentOuter: {
+        height: '100%',
+        display: 'flex',
+        flexDirection: 'row',
+        justifyContent: 'center',
+    },
+    contentInner: {
+        width: '100%',
+    },
```

Comment:
Did you add content/contentOuter/contentInner back in explicitly? I removed them in another PR because they were unnecessary and the styles should be handled within the children components, but if you need them, it's fine, just want to make sure it's not from an outdated merge

Reply:
not intentional. I will remove these.